docs/src/pages/layout/jss/MediaQuery.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from 'react';
+import PropTypes from 'prop-types';
+import compose from 'recompose/compose';
+import { withStyles, createStyleSheet } from 'material-ui/styles';
+import withWidth from 'material-ui/utils/withWidth';
+import Typography from 'material-ui/Typography';
+
+const styleSheet = createStyleSheet('MediaQuery', theme => ({
+  root: {
+    position: 'relative',
+    padding: '10px',
+  },
+  [theme.breakpoints.up("md")]: {
+    root: {
+      backgroundColor: theme.palette.primary[500],
+    },
+  },
+  [theme.breakpoints.down("md")]: {
+    root: {
+      backgroundColor: theme.palette.accent[500],
```

Comment:
For 90% for the situations, the material specification ask for using the `A` color with the accent.
`backgroundColor: theme.palette.accent.A400,`